actionpack/lib/action_dispatch/middleware/ssl.rb

```diff
@@ -34,6 +34,13 @@ module ActionDispatch
   # original HSTS directive until it expires. Instead, use the header to tell browsers to
   # expire HSTS immediately. Setting `hsts: false` is a shortcut for
   # `hsts: { expires: 0 }`.
+  #
+  # Redirection can be constrained to only some requests by setting `constrain_to`:
+  #
+  #    config.ssl_options = { redirect: { constrain_to: -> request { request.path !~ /healthcheck/ } } }
+  #
+  # If set, requests are only redirected if the `constrain_to` block returns true.
+
```

Comment:
I ended up updating the suggested documentation a few times, can you use the latest revision?

(We also don't need the blank line after the documentation comments.)